### DISCUSSION TITLE
Implement SMC^2

### DISCUSSION THREAD

**Discussion by rlouf** [blackjax-devs/blackjax#88] [New features]
https://arxiv.org/pdf/1101.1528.pdf

**Comment by AdrienCorenflos** [comment]
Is there a specific reason why you'd want this algorithm implemented? It is very SSM oriented IMO (though @nchopin may have a different point of view).

**Comment by rlouf** [comment]
That's an algorithm I've wanted to implement for a while and that would be very useful for many practitioners.
